robolectric/src/main/java/org/robolectric/android/controller/IntentServiceController.java

```diff
@@ -20,47 +13,11 @@
                                                                         final T service,
                                                                         final Intent intent) {
     final IntentServiceController<T> controller = new IntentServiceController<>(shadowsAdapter, service, intent);
-      controller.attach();
       return controller;
     }
 
     private IntentServiceController(final ShadowsAdapter shadowsAdapter, final T service, final Intent intent) {
         super(shadowsAdapter, service, intent);
-        shadowActivityThreadClassName = shadowsAdapter.getShadowActivityThreadClassName();
-    }
-
-  /**
-   * @deprecated This is a no-op, it's safe to remove this call.
-   *
-   * This method will be removed in Robolectric 3.4.
-   */
-  @Deprecated
-  public IntentServiceController<T> attach() {
```

Comment:
As above -- this method can be made private but still needs to be called from create(). (Sorry, the deprecation comment was misleading).